appveyor.yml

```diff
@@ -6,54 +6,35 @@ matrix:
 environment:
   matrix:
 
-    # Pre-installed Python versions, which Appveyor may upgrade to
+    # Pre-installed Python versions, which AppVeyor may upgrade to
     # a later point release.
     # See: http://www.appveyor.com/docs/installed-software#python
 
-    # Python 2.7.11 is the latest Python 2.7 with a Windows installer
-    # Python 2.7.11 is the overall latest
-    # https://www.python.org/ftp/python/2.7.11/
     - PYTHON: "C:\\Python27"
-      PYTHON_VERSION: "2.7.11"
-      PYTHON_ARCH: "32"
```

Comment:
@jonparrott These env. vars are used by many helpers e.g. the common `CMD_IN_ENV` helpers, but they are largely irrelevant if you never need the Windows SDK. (They were in theory being using by `install.ps1`, but in practice never used.)